OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,77 +1,11 @@
 SOA on Docker
 =============
 
-Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for SOA 12.2.1.2.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle WebLogic Infrastructure 12.2.1.2.0.
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for SOA 12.2.1.x based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle WebLogic Infrastructure 12.2.1.x.
```

Comment:
It's based on Fusion Middleware, not WebLogic.

Reply:
Fixed it.